ChainReactionTests/Registration.feature

```diff
@@ -21,10 +21,30 @@ Scenario:  Verify Register
 	Then I See My Account page is loaded
 
 Scenario: Verify Login and Logout
-	Given Click on Logout button
-	And See that i navigate to the site
-	When Click on SignIn link
-	And See that i navigate to the SignIn page
-	And Fill login and password fields
-	Then Click on SignIn button
-	And See that i login
\ No newline at end of file
+	Given I start the browser
+	When I Navigate to the url "http://www.chainreactioncycles.com/ru/en"
+	And I See the main page is loaded
+
+	When I Click on RubRub dropdown
+	And I select "English" language and click Update button
+	Then I See the main page is loaded
+
+	When I Click on Sign In on top menu
+	Then I See the Sign In page is loaded
+
+	When I set following parameters on Sign In page
+		| Field | Value |
+		| email  |   |
+		| pass  |   |
+	And I Click Sign In button
+	Then I See the message on Sing In page

```

Comment:
Need to add the message text so that step will look like following:
Then I See the message with text "blah-blah-blah" on Sing In page